CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+demisto_sdk.tests.integration_tests.content_create_artifacts_integration_test
```

Comment:
```suggestion
Fixed an issue where the **doc-review** command was successful even when the release-note is malformed.
```
not sure what is your current change log here?

Reply:
bad merge, sorry